js/spreadsheet.js

```diff
@@ -225,6 +225,22 @@ var spreadsheet = function(options) {
     },
     afterSelectionEnd: function(r, c, r2, c2) {
       s = [r, c, r2, c2];
+    },
+    beforeKeyDown: function (event) {
+      event = event || window.event;
+
+      if ((event.ctrlKey || event.metaKey) && event.keyCode === 65 ) {
+        event.stopImmediatePropagation();
+        
+        var colEnd = getColumns().filter(function(column) {
+          return column;
+        }).length - 1;
```

Comment:
What if `.length` returns `0`?

Reply:
@tonytlwu 
This scenario is possible only if the user has absolutely empty data source without a single filled row.
And even when a user creates a new one data source it already has default values.
Should we add if condition to check for this scenario?

Reply:
My philosophy in error prevention is that when programming, the developer should try to not think about the user scenario but about the possible states of the variable. It isn't useful for developers to say, "well I didn't think users would do XYZ" when a bug occurs.

So yes, we should add conditions to check for this scenario.

Reply:
Also, be sure to resolve the conflict before the next review.